plugin/adsk/plugin/exportTranslator.cpp

```diff
@@ -52,6 +52,12 @@ UsdMayaExportTranslator::writer(const MFileObject &file,
                  const MString &optionsString,
                  MPxFileTranslator::FileAccessMode mode ) {
 
+    // If we are in neither of these modes then there won't be anything to do
+    if (mode != MPxFileTranslator::kExportActiveAccessMode && 
+        mode != MPxFileTranslator::kExportAccessMode) {
+        return MS::kSuccess;
+    }
```

Comment:
The logic farther down used to be that we would only fill the input list if one of these two flags were set.  If neither were set then we had nothing to export and just returned kSuccess.  Moving this check to the start since there is no point in continuing if we are in a different mode.